standards/rfc7030/src/lib.rs

```diff
@@ -0,0 +1,206 @@
+/*
+From RFC7030 (https://datatracker.ietf.org/doc/html/rfc7030):
+CsrAttrs ::= SEQUENCE SIZE (0..MAX) OF AttrOrOID
+AttrOrOID ::= CHOICE { oid OBJECT IDENTIFIER, attribute Attribute }
+Attribute ::= SEQUENCE {
+        type  OBJECT IDENTIFIER,
+        values SET SIZE(0..MAX) OF ANY }
+*/
+
+#[cfg(test)]
+type CsrAttrs = Vec<AttrOrOid>;
+
+#[derive(rasn::AsnType, rasn::Decode, rasn::Encode, Debug, PartialEq, Clone)]
+#[rasn(choice)]
+enum AttrOrOid {
+    OID(rasn::types::ObjectIdentifier),
+    ATTRIBUTE(Attribute),
+}
+
+#[derive(rasn::AsnType, rasn::Decode, rasn::Encode, Debug, PartialEq, Clone)]
+struct Attribute {
+    r#type: rasn::types::ObjectIdentifier,
+    values: rasn::types::SetOf<rasn::types::Open>,
```

Comment:
This should be using `Any` not `Open`. `Open` can't handle things like sequences, where as `Any` can.